src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -216,6 +218,60 @@ Use the codelab aside to link to an associated codelab.
   </button>
 </div>
 
+## Callouts
+
+{% CodelabsCallout ['codelab-fix-sneaky-404', 'codelab-art-direction', 'codelab-cloudinary'] %}
+
+{% SelfAssessment 'Use the drop-down below each code sample to check whether it needs ARIA information.' %}
+{% Tabset 'Samples for knowledge self check' %}
+
+{% Tab 'sample' %}
+
+```html
+<label for="pwd-input">Password</label>
+
+<input type="text" role="textbox" id="pwd-input" name="password">
+```
+
+{% SelfAssessmentHint 'Does the sample need ARIA?' %}
+**No.** This sample is **incorrect**.
```

Comment:
Well, it adds a `role` attribute to the text input. I thought you weren't supposed to that?

Reply:
Based on the fix I made to the last sentence in the answer and the summary at the top of the [MDN textbox role page](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/textbox_role), I'm gonna resolve this so I don't hold up Ewa. Happy to change the example later if it's still off; I figure it's not a huge deal since it just shows up in the handbook.

Reply:
ah i see